server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -1306,6 +1313,10 @@ public void endVerification(String seed) {
     private final AtomicReference<CachedRepositoryData> latestKnownRepositoryData =
             new AtomicReference<>(new CachedRepositoryData(RepositoryData.EMPTY_REPO_GEN, null));
 
+    protected BlobContainer testBlobContainer(String seed) {
+        return blobStore().blobContainer(basePath().add(testBlobPrefix(seed)));
+    }
```

Comment:
I introduced this because I wanted the blob container for test blobs to bypass encryption, otherwise repository verification has to create blob generations which it can't if the repo is read-only.